### PR TITLE
Add document for adding version config in PostgreSQL adapter

### PR DESCRIPTION
**Pull request by RockyNiu**
Add the document for the changes discussed in https://github.com/tgriesser/knex/pull/1959

### CODE REVIEW

File: sections/installation.js [hunk 1]
```diff
@@ -102,6 +102,26 @@ export default [
...
+    type: "info",
+    content: "Note: The database version can be added in knex configuration, when you use the PostgreSQL adapter to connect a non-standard database."
+  },
+  {
+    type: "code",
+    language: "js",
+    content: `
+      var knex = require('knex')({
+        client: 'pg',
+        version: 'v7.2.1-0',
```
Comment (elhigu) [inline sections/installation.js]: I think this is wrong... it should be something like e.g. `9.5` to be able to select which postgres features will be available. Now this example looks like you are giving the string that is parsed from the version string provided by custom database.

Comment (RockyNiu) [inline sections/installation.js]: En. Now I see what you mean. Fixed.

**Comment by elhigu** [comment]
looks good now, thanks!

**Comment by RockyNiu** [comment]
no problem :)

**Pull request merged**